### PR TITLE
Adding existing governance materials

### PR DESCRIPTION
**Pull request by ellisonbg**
This PR adds the existing governance document from the IPython wiki. I will put a link on that wiki page to this repo as well.

Once this PR is merged I will submit further PRs to clean up this material and start to add new things like the incubator.

@jupyter/steeringcouncil

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
-# governance
-The governance process and model for Project Jupyter
+# Project Jupyter Governance
+
+The purpose of this repository is to formalize the governance process that the IPython/Jupyter project has used informally since its inception in 2001. This document clarifies how decisions are made and how the various elements of our community interact, including the relationship between open source collaborative development and work that may be funded by for-profit or non-profit entities.
```
Comment (Carreau) [inline README.md]: can you wrap these line if ever we have to change them ?

File: governance.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
...ucible, exploratory and interactive computing. The Software developed by The Project is released under the BSD (or similar) open source license, developed openly and hosted on public GitHub repositories under the ipython and jupyter GitHub organizations. Examples of Project Software include the IPython Notebook, the IPython Terminal, IPython.parallel, the Jupyter Hub, etc. The Services run by the Project consist of public websites and web-services that are hosted under the jupyter.org or ipython.org domains. The Services Examples of Project Services include the Jupyter and IPython websites ([http://jupyter.org](http://www.google.com/url?q=http%3A%2F%2Fjupyter.org&sa=D&sntz=1&usg=AFQjCNHztfVQq8SN6qni-DVgmoaTYBVasg) and [http://ipython.org](http://www.google.com/url?q=http%3A%2F%2Fipython.org&sa=D&sntz=1&usg=AFQjCNGiroA4bzjI-kCsgV4YdTPbgyuyeA)), nbviewer ([https://nbviewer.ipython.org](https://www.google.com/url?q=https%3A%2F%2Fnbviewer.ipython.org&sa=D&sntz=1&usg=AFQjCNGB-iNba6jJiu1N9Oo4oMTWwFq3Aw)) and the Jupyter coLaboratory ([https://colaboratory.jupyter.org](https://www.google.com/url?q=https%3A%2F%2Fcolaboratory.jupyter.org&sa=D&sntz=1&usg=AFQjCNFBxEodelaV8P877UGqxrp7WU8Y3g)).
```
Comment (Carreau) [inline governance.md]: What are these weird google URL ?

File: governance.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+**Tier 2** = an institution with at least one Institutional Contributor
+
+-   Same benefits as Tier 1 level Partners, but:
+-   Only Institutional Contributors are invited to the Institutional Partners Workshop and bi-annual Jupyter/IPython Developer Meeting.
+
+Existing **Tier 2** Partners:
+
+-   Bloomberg (Jason Grout, Sylvain Corlay)
+-   Google (Kester Tong, Kayur Patel, Mark Sandler)
```
Comment (Carreau) [inline governance.md]: Need update IBM and Continuum ?

File: governance.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+-   Ability to influence the project through the participation of their Council Member.
+-   Council Members invited to bi-annual Jupyter/IPython Developer Meeting.
+
+Existing **Tier 1** level Institutional Partners:
+
+-   UC Berkeley (Fernando Pérez, Min Ragan-Kelley, Thomas Kluyver)
+-   Cal Poly (Brian Granger)
+-   Rackspace (Kyle Kelley)
+-   Continuum Analytics (Damian Avila)
+
```
Comment (damianavila) [inline governance.md]: (Bloomberg) Jason Grout should be here as well...
I guess also Thomas and MinRK's new institutions, but not sure if the association is immediate or there is a "buffer" time...

File: governance.md (hunk 3 above)
Comment (damianavila) [inline governance.md]: yep, probably... but I guess @ellisonbg wants this one merged as is and then start making the corrections... I am ok with that if that's the case...

File: governance.md (hunk 2 above)
Comment (rgbkrk) [inline governance.md]: Looks like copying of links from a google search page.

Comment (ellisonbg) [inline governance.md]: Hi all, thanks for the feedback. However, to have a full record of the
evolution of the document, I will like to merge it basically as is and then
start to fix the many things that need updates in further PRs.

On Wed, Sep 2, 2015 at 7:01 AM, Kyle Kelley <notifications@github.com>
wrote:

> In governance.md
> <https://github.com/jupyter/governance/pull/1#discussion_r38534589>:
>
> > @@ -0,0 +1,136 @@
> > +# Main Governance Document
> > +
> > +The Project
> > +===========
> > +
> > +The Jupyter/IPython Project (The Project) is an open source software project affiliated with the 501c3 NumFocus Foundation. The goal of The Project is to develop open source software and deploy open and public websites and services for reproducible, exploratory and interactive computing. The Software developed by The Project is released under the BSD (or similar) open source license, developed openly and hosted on public GitHub repositories under the ipython and jupyter GitHub organizations. Examples of Project Software include the IPython Notebook, the IPython Terminal, IPython.parallel, the Jupyter Hub, etc. The Services run by the Project consist of public websites and web-services that are hosted under the jupyter.org or ipython.org domains. The Services Examples of Project Services include the Jupyter and IPython websites ([http://jupyter.org](http://www.google.com/url?q=http%3A%2F%2Fjupyter.org&sa=D&sntz=1&usg=AFQjCNHztfVQq8SN6qni-DVgmoaTYBVasg) and [http://ipython.org](http://www.google.com/url?q=http%3A%2F%2Fipython.org&sa=D&sntz=1&usg=AFQjCNGiroA4bzjI-kCsgV4YdTPbgyuyeA)), nbviewer ([https://nbviewer.ipython.org](https://www.google.com/url?q=https%3A%2F%2Fnbviewer.ipython.org&sa=D&sntz=1&usg=AFQjCNGB-iNba6jJiu1N9Oo4oMTWwFq3Aw)) and the Jupyter coLaboratory ([https://colaboratory.jupyter.org](https://www.google.com/url?q=https%3A%2F%2Fcolaboratory.jupyter.org&sa=D&sntz=1&usg=AFQjCNFBxEodelaV8P877UGqxrp7WU8Y3g)).
>
> Looks like copying of links from a google search page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/governance/pull/1/files#r38534589>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

Comment (rgbkrk) [inline governance.md]: Did you mean to pull in the commits for just this document then? This isn't a full record at all.

```
~/code/src/github.com/ipython/ipython.wiki$ git log IPEP-29\:-Project-Governance.md
commit c99985dbeec1be7b1ecd43ca280d1516e7e80187
Author: Brian E. Granger <ellisonbg@gmail.com>
Date:   Tue Sep 1 22:54:23 2015 -0700

    Silly small stuff.

commit 33f4694dbd1fcd5ea837aa15b934ec87596e07e6
Author: Brian E. Granger <ellisonbg@gmail.com>
Date:   Tue Sep 1 22:54:01 2015 -0700

    Adding note to new jupyter/governance repo.

commit c5ac798ac2e107076360beefff488ce6c6533bc5
Author: ellisonbg <ellisonbg@gmail.com>
Date:   Tue Nov 4 12:38:48 2014 -0800

    Adding Mark Sandler

commit f26042d4070a0c9cb57759709d1094ef0a743a03
Author: fperez <fperez.net@gmail.com>
Date:   Fri Oct 31 10:53:04 2014 -0700

    Uniform terminology for 'non-profit'.

commit 331e06d852d6637466e9012cf9eb0d39def6a3a6
Author: ellisonbg <ellisonbg@gmail.com>
Date:   Mon Oct 27 17:01:40 2014 -0700

    Adding Ins. Partner approval and grace period.

commit 543d133a9e3b5092a8987424d65d565c4f8585cf
Author: ellisonbg <ellisonbg@gmail.com>
Date:   Mon Oct 27 16:56:31 2014 -0700

    Changing BDFL succession process.

commit d00be72bfeda445c8bde738bd4578f9420ba096b
Author: ellisonbg <ellisonbg@gmail.com>
Date:   Mon Oct 27 16:51:09 2014 -0700

    Gold/Silver -> Tier 1/2

commit 59c712083cda75e02a486d91e5f95b1acede9211
Author: takluyver <takowl@gmail.com>
Date:   Fri Oct 24 15:09:24 2014 -0700

    Created IPEP 29: Project Governance (markdown)
```

**Pull request merged**